### ISSUE TITLE
exclude only parts of rule in cli

### ISSUE THREAD

**Issue by kkmuffme** [squizlabs/PHP_CodeSniffer#2313]
I want to exclude 2 specific errors from a rule, not the complete rule.

This doesn't work, I can only exclude the complete rule:
<exclude name="Squiz.PHP.EmbeddedPhp.ContentBeforeOpen"/>
<exclude name="Squiz.PHP.EmbeddedPhp.ContentBeforeEnd"/>

This works when using a custom xml ruleset.

This doesn't work when using the --exclude option in cli with error:
ERROR: The specified sniff code "Squiz.PHP.EmbeddedPhp.ContentBeforeEnd" is invalid

**Comment by jrfnl** [comment]
Excludes need to be nested in a `<rule>`, so (based on your other issue), if you are getting these issue because you include one of the WPCS rulesets, try this:
```xml
<rule ref="WordPress">
    <exclude name="Squiz.PHP.EmbeddedPhp.ContentBeforeOpen"/>
    <exclude name="Squiz.PHP.EmbeddedPhp.ContentBeforeEnd"/>
</rule>
```

As for the command line: the `exclude` option there only allows for ignoring complete sniffs, not individual error codes.

**Comment by kkmuffme** [comment]
Ok clear. Thanks!
